### PR TITLE
Food cubes

### PR DESCRIPTION
**Pull request by Ketrai**
Adds food cubes to provide more variety when it comes to nutritious food sources.

- On par with protein and liquid food packs.
- Worst taste of any food in the game.
- Adds a vending machine in medical that dispenses 5 trays (5 more if hacked)
- Adds a version with 40 trays and 20 extra when hacked for use in missions, pois, etc.
- Poisonous raw, add water to get a protein/nutrient slab.
- Can be ground down and combined with virus food to create 60 protein or 60 nutrient respectively, making it easier to use for medical pill creation than protein shakes. Also means you can refill shakers, or fill empty shakers more easily than before. 
- Tiny cubes that come as 8 in a small tray. This means you can take extreme amounts of food to expiditions relatively easy. 
- 10 trays can be bought from cargo for 75, making it an expensive investment that should be able to feed 10 crew members for an entire shift.
- rewrote chicken basket code so it can be used for any kind of custom food box. ideally we rewrite it to make use of the already existing icon update code for donut boxes etc eventually.

I didn't touch protein shakes yet, but at 35 protein, 30 nutrients, they should probably be considered for a nerf eventually. Because of the nature of cubes needing water/virus food to expand, they could be used to fill up empty-bought shakers instead. Or otherwise lengten shakers if they didn't spawn with an obscene amount already. These are just suggestions though, as it seems most people prefer the easy route of ready-made shakes, and being good for an entire shift on just two of the things. Feels a little too easy.

### CODE REVIEW

**Comment by Ketrai** [comment]
Also I have noticed the map diffs being weird as hell, I only changed one tile. I ran dmm2tgm.bat but it seems to have flipped out, anything up with the map merge?

File: code/modules/food/food/snacks_vr.dm [hunk 1]
```diff
@@ -678,3 +681,85 @@
...
+	name = "Nutriment slab"
+	desc = "A slab of near pure plant-based nutrients, extremely artificial, and thoroughly disgusting."
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "nutrimentslab"
+	bitesize = 10
+	nutriment_amt = 20
+	nutriment_desc = list("compost" = 50)
+
+/obj/item/weapon/reagent_containers/food/snacks/nutrimentslab/Initialize()
+	..()
```
Comment (Screemonster) [inline code/modules/food/food/snacks_vr.dm]: Shouldn't there be an add reagent here like the protein one?

Comment (Ketrai) [reply code/modules/food/food/snacks_vr.dm]: Nope, that's all in the nutriment_amt. It's just a pure block of 40 nutriment. It does however look like this line isn't needed.

**Comment by Ketrai** [comment]
fixed the weird mapmerge error.

**Comment by lbnesquik** [comment]
I think the cargo order should be cheaper.

**Comment by Ketrai** [comment]
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE APPARENTLY GIT CHECKOUT REBASED ONTO MY OWN MASTER OR SOME SHIT AND I COPIED THE WRONG FILE TO REPLACE THE TETHER STATION MAP DMM TO RESOLVE CONFLIIIIIIICTS.

I DON'T HAVE TIME FOR THIS

PLEASE HALP.

**Comment by TheFurryFeline** [comment]
Working on fixing your conflicts. Be ready to check your fork's branch for a PR. Additionally, this apparently will be resolving an issue I posted in December. Please insert this keyword, `resolves #6063` so it closes automatically when this gets merged. :>

**Comment by TheFurryFeline** [comment]
https://github.com/Ketrai/VOREStation/pull/1 PR up, though not sure since checks aren't starting on the fork. But it should work.

**Comment by Ketrai** [comment]
Moved to  #6404

**Pull request closed without merging**